geode-protobuf/src/main/java/org/apache/geode/internal/protocol/protobuf/statistics/ProtobufClientStatisticsImpl.java

```diff
@@ -35,6 +37,9 @@
   private final int authenticationFailuresId;
 
   public ProtobufClientStatisticsImpl(StatisticsFactory statisticsFactory, String statisticsName) {
+    if (statisticsFactory == null) {
```

Comment:
@upthewaterspout @PivotalSarge I'm curious that this can happen. I thought we'd put logic in so that the statistics initialization would be separate. However, looking at `InternalLocator.myDS` it looks like the variable may be null... we should fix that at some point.